.sync.yml

```diff
@@ -53,25 +46,29 @@ Gemfile:
           - mswin
           - mingw
           - x64_mingw
-      - gem: beaker
-        version: '~> 3.13'
-        from_env: BEAKER_VERSION
-      - gem: beaker-abs
-        from_env: BEAKER_ABS_VERSION
-        version: '~> 0.1'
-      - gem: beaker-pe
-      - gem: beaker-hostgenerator
-        from_env: BEAKER_HOSTGENERATOR_VERSION
-      - gem: beaker-rspec
-        from_env: BEAKER_RSPEC_VERSION
+
+spec/spec.opts:
+  delete: true
+
+spec/spec_helper_local.rb:
+  delete: true
 
 .project:
   delete: true
 
+locales/config.yaml:
```

Comment:
Do we need this? This module does not have any locales

Reply:
Removed